Update other connector samples (#261)
SteeltoeOSS/Samples: Connectors/src/SqlServerEFCore/README.md

﻿# SQL Server Connector Sample App - Entity Framework Core

> ASP.NET Core sample app illustrating how to use Entity Framework Core together with [Steeltoe SQL Server Connector](https://github.com/SteeltoeOSS/Connectors) for connecting to a SqlServer service on CloudFoundry.

Reviewer: ```suggestion
ASP.NET Core sample app illustrating how to use Entity Framework Core together with [Steeltoe SQL Server Connector](https://docs.steeltoe.io/api/v3/connectors/microsoft-sql-server.html) for connecting to a SQL Server service on CloudFoundry.
```

## General Pre-requisites

1. Installed .NET Core SDK

## Running Locally

1. Installed SQL Server
1. Created SQL Server database and user with appropriate access level
1. Set [ASPNETCORE_ENVIRONMENT=Development] (<https://docs.microsoft.com/en-us/aspnet/core/fundamentals/environments>)
1. Added your connection string to appsettings.development.json under SqlServer:Credentials:ConnectionString

## Running on CloudFoundry

1. Installed Pivotal CloudFoundry
1. (Optional) installed Windows support
1. A SQL Server Instance

> 1. (Optional) Installed [Microsoft SQL Server broker](https://github.com/cf-platform-eng/mssql-server-broker)

Reviewer: ```suggestion
1. (Optional) Installed [Cloud Service Broker for VMware Tanzu](https://docs.vmware.com/en/Cloud-Service-Broker-for-VMware-Tanzu/index.html)
```

---

> ### See the Official [Steeltoe Service Connectors Documentation](https://steeltoe.io/docs/steeltoe-service-connectors) for a more in-depth walkthrough of the samples and more detailed information.

Reviewer: ```suggestion
### See the Official [Steeltoe Service Connectors Documentation](https://docs.steeltoe.io/api/v3/connectors/) for a more in-depth walkthrough of the samples and more detailed information.
```
